ext/devcontainer/src/radcli/devcontainer-feature.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Radius CLI",
+    "id": "radcli",
+    "version": "0.1.0",
```

Comment:
@willdavsmith - it looks like this file hardcodes versions. This will be another thing for us to automate as part of the release.

Reply:
I don't want this to block the PR, but it's something to consider.